packages/odyssey-react/.storybook/decorators/withGlobalReset.tsx

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ComponentType } from 'react';
+
+export function withGlobalReset(Story: ComponentType) {
+  return (
+    <>
+      <style>{require('!to-string-loader!css-loader!@okta/odyssey/src/scss/base/_reset.scss')}</style>
```

Comment:
This is inserted _after_ our non-reset styles, so it's overriding properties like `font-family` even when those declarations are presented earlier.